plotly/tools.py

```diff
@@ -1670,6 +1722,130 @@ def create_streamline(x, y, u, v,
                              mode='lines', **kwargs)
         return streamline
 
+    @staticmethod
+    def create_ohlc(open, high, low, close, direction, **kwargs):
+
```

Comment:
:cow2: I don't believe we typically leave a space between a function and the docstring.